inv/inventory.go

```diff
@@ -17,9 +17,11 @@ package inv
 import (
 	"context"
 
+	"github.com/mendersoftware/go-lib-micro/log"
```

Comment:
at least let's move it to the rest of mendersoftware ones.

Reply:
Done